src/main/java/net/geforcemods/securitycraft/tileentity/TrophySystemTileEntity.java

```diff
@@ -83,8 +89,7 @@ private Entity getTarget() {
 
 		// Add all arrows and fireballs to the targets list. Could always add more
 		// projectile types if we think of any
-		potentialTargets.addAll(world.getEntitiesWithinAABB(ArrowEntity.class, area));
```

Comment:
Why did you remove this?

Reply:
I thought that Arrows are also a subclass from DamagingProjectiles, my bad

Reply:
I didn't mark that line with my comment, be more careful of that, please.